### DISCUSSION TITLE
Allow background color option for dark themed IDE's

### DISCUSSION THREAD

**Discussion by aastopher** [shap/shap#2566] [Ideas]
VS code dark theme makes plots difficult to read. Ideally force a white background by default. However, if we could just define a background color that would work well enough.

![vscode_dark_theme](https://user-images.githubusercontent.com/22336995/171061959-665347c1-7d3c-4cc5-9b98-6d99fd3dea8f.png)

**Comment by neiljdo** [comment]
Hi @aastopher!

I have exactly the same issue. I usually prefer dark themes with JupyterLab and SageMaker but the transparent background for some of the plots forces me to switch to a light theme.

**Comment by AlanBlanchet** [comment]
Are there any updates to this ? 

![image](https://user-images.githubusercontent.com/22567306/220970103-96ec7ece-0f0e-4dbc-9a6f-6198faa717c6.png)

EDIT : Found [this](https://youtu.be/9haIOplEIGM?list=PLV8yxwGOxvvovp-j6ztxhF3QcKXT6vORU&t=574) Youtuber having to change from dark to white mode in vscode because of this

**Comment by AlanBlanchet** [comment]
Found a solution by wrapping the shap html into a div with a **background color of my choice**. Here is my code :
```python
import shap
from IPython.display import HTML

shap.initjs()

explainer = shap.Explainer(emission_models[3])
shap_values = explainer(emissionTTS[0])

exp = shap.Explanation(shap_values.values, shap_values.base_values[0][0], shap_values.data)

force_plot = shap.force_plot(exp)
HTML(f"<div style='background-color:white;'>{shap.getjs() + force_plot.html()}</div>")
```

![image](https://user-images.githubusercontent.com/22567306/221136132-e2521ace-7b0e-4449-ad8c-7400525c6aad.png)
